src/MoBi.Assets/AppConstants.cs

```diff
@@ -1969,7 +1969,11 @@ public static string MoleculeNodeAlreadyExistsForMolecule(string moleculeName)
 
       public static string RHSDefaultUnitName(IDimension dimension)
       {
-         return $"{dimension.BaseUnit.Name}/min";
+         var numerator = string.IsNullOrEmpty(dimension.BaseUnit.Name) ? "1" : dimension.BaseUnit.Name;
```

Comment:
Such as stupid mistake here. The unit returned was "/min" instead of "1/min"!
Same with min/min => I return string.empty